components/teams-service/storage/postgres/migration/sql/10_upgrade_to_v2_migrations.sql

```diff
@@ -0,0 +1,23 @@
+BEGIN;
+
+UPDATE teams
+    SET
+        name = 'editors',
+        description = 'Editors'
+    WHERE
+        name = 'operators'
```

Comment:
should we also set the team's projects to empty too, like in the old 2nd data migration?

Reply:
I don't think we need to do it twice-- I couldn't think of a scenario in which they could have had their projects dropped, then added more, because I think we enforced permissions in the meantime. 

But if you can think of one, we can do it!